pilot/pkg/serviceregistry/serviceentry/servicediscovery_test.go

```diff
@@ -672,7 +672,10 @@ func TestServiceDiscoveryWorkloadUpdate(t *testing.T) {
 		}
 		expectProxyInstances(t, sd, instances, "2.2.2.2")
 		expectServiceInstances(t, sd, selector, 0, instances)
-		expectEvents(t, events, Event{kind: "eds", host: "selector.com", namespace: selector.Namespace, endpoints: 2})
+		expectEvents(t, events,
+			Event{kind: "xds", proxyIP: "2.2.2.2"},
```

Comment:
why does this change?

Reply:
```
		// trigger full xds push to the related sidecar proxy
		if event == model.EventAdd {
			s.XdsUpdater.ProxyUpdate(s.Cluster(), wle.Address)
		}
```

Previously, this event is ignored, if you run the test without this pr,  should see the ignore log. But now eds event is sent asynchrously, so i make it happen after proxy update event

Reply:
:+1: